chain/src/chain.rs

```diff
@@ -857,6 +857,14 @@ impl Chain {
 		}
 
 		let header = self.get_block_header(&h)?;
+
+		{
+			let mut txhashset_ref = self.txhashset.write();
+			// Drop file handles in underlying txhashset
+			txhashset_ref.release_backend_files();
```

Comment:
Seems to work, would just like a bit of reassurance that this isn't going to have any odd side effects.

Reply:
Not that I can think of, seems fair.